docs/source/developers/python.rst

```diff
@@ -724,26 +724,18 @@ Installing Nightly Packages
 .. warning::
     These packages are not official releases. Use them at your own risk.
 
-PyArrow has nightly wheels and Conda packages for testing purposes.
+PyArrow has nightly wheels for testing purposes hosted at
+`scientific-python-nightly-wheels
+<https://anaconda.org/scientific-python-nightly-wheels/pyarrow>`_.
 
 These may be suitable for downstream libraries in their continuous integration
 setup to maintain compatibility with the upcoming PyArrow features,
-deprecations and/or feature removals.
+deprecations, and/or feature removals.
 
-Install the development version of PyArrow from `arrow-nightlies
-<https://anaconda.org/arrow-nightlies/pyarrow>`_ conda channel:
+To install the most recent nightly version of PyArrow, run:
 
 .. code-block:: bash
 
-    conda install -c arrow-nightlies pyarrow
-
-Note that this requires to use the ``conda-forge`` channel for all other
-packages (``conda config --add channels conda-forge``).
-
-Install the development version from an `alternative PyPI
-<https://gemfury.com/arrow-nightlies>`_ index:
-
-.. code-block:: bash
-
-    pip install --extra-index-url https://pypi.fury.io/arrow-nightlies/ \
-        --prefer-binary --pre pyarrow
+    pip install \
+      -i https://pypi.anaconda.org/scientific-python-nightly-wheels/simple \
+      pyarrow
```

Comment:
You don't need the `--pre` to get it actually from the nightly index?

Reply:
It doesn't seem to be required:
```
$ pip install -i https://pypi.anaconda.org/scientific-python-nightly-wheels/simple pyarrow
Looking in indexes: https://pypi.anaconda.org/scientific-python-nightly-wheels/simple
Collecting pyarrow
  Downloading https://pypi.anaconda.org/scientific-python-nightly-wheels/simple/pyarrow/20.0.0.dev4/pyarrow-20.0.0.dev4-cp312-cp312-manylinux_2_28_x86_64.whl (42.2 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 42.2/42.2 MB 13.2 MB/s eta 0:00:00
Installing collected packages: pyarrow
Successfully installed pyarrow-20.0.0.dev4
```